index.html

```diff
@@ -705,6 +719,21 @@ <h3>
         <h3>
           Starting a new presentation example
         </h3>
+        <p>
+          When the user clicks <code>presentBtn</code>, this code requests
+          presentation of one of the URLs in the <a>PresentationRequest</a>.
+          When <a data-link-for="PresentationRequest">start</a> is called, the
+          browser typically shows a dialog listing compatible displays and
```

Comment:
It's slightly unclear to me whether "pick one" refers to the display or the URL. Suggestion: "... typically shows a dialog that allows the user to select one of the compatible displays that are available"

Reply:
Done